### ISSUE TITLE
为什么推理步骤的pos_embed是通过gt_mask得到的，gt_mask参与推理是否属于信息泄漏？

### ISSUE THREAD

**Issue by ybhwq** [lifier/LMAFormer#11]
推理阶段的[pos_embed](https://github.com/lifier/LMAFormer/blob/main/MFIRSTD/models/lmaf_swin.py#L973) 指向的是[PositionEmbeddingSine](https://github.com/lifier/LMAFormer/blob/main/MFIRSTD/models/position_encoding.py#L11-L60)，编码过程的输入好像是gt_mask。推理过程中混合使用了gt_mask是否属于信息泄露？

**Comment by lifier** [comment]
您说的很对，当时写代码的时候没注意，训练过程可以用gt，推理忘记改了，可在pos_embed部分添加如下代码：
if self.training:
    mask = tensor_list.mask
else:
    mask = generate_inference_mask(tensor_list.tensors)

def generate_inference_mask(tensors):
     return tensors == 0
